.github/workflows/daily_jupyter_nb_test.yml

```diff
@@ -65,7 +65,7 @@ jobs:
                   - image_name: windows-latest
                     download_name: windows
                 python-version: ["3.9", "3.10", "3.11"]
-                cloud-provider: [aws, azure, gcp]
+                cloud-provider: [aws]
```

Comment:
Do we want to limit the test to just AWS?

Reply:
The FINANCIAL__ECONOMIC_ESSENTIALS.CYBERSYN database used for the Snowpark Pandas demo wasn't available in azure or gcp so I had to limit the test to aws for now

Reply:
could you add a comment about that?

i doubt we need to test on the other cloud providers anyway, but it's good for future reference to have that information here.

Reply:
add a comment in the yaml file? Sure